PlayerCore/components/AdVASTModel.swift

```diff
@@ -3,15 +3,15 @@
 
 import Foundation
 extension Ad {
-    public struct VASTModel: Equatable {
+    public struct VASTModel: Hashable {
```

Comment:
Why you are changing everything from Equatable to Hashable?

Reply:
I thought that only `Hashable` could be stored as a value into Dictionary :)

Reply:
So `Equatable` not needed?

Reply:
No `Equatable` needed for tests

Reply:
Got it.